src/main/resources/userEmailTemplate-feedbackSessionClosing.html

```diff
@@ -60,8 +60,16 @@
 </p>
 
 <p>
-    <strong>No action is required if you have already submitted.</strong>
-    <br>If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
+    <ul>
+        <li><strong>If already submitted:</strong></li>
+        No action is required.
+        <li><strong>Requests for deadline extensions,</strong></li>
+        <strong>problems regarding team/student data</strong> (e.g. wrong team, misspelled name),
+        <br>and <strong>other non-technical queries</strong> about the feedback session :
+        <br>Contact the instructors of the course: Insructor Name (instructor.email@example.com).
```

Comment:
Instead of hard coding in a value for the instructor name and email, you will have to use `${coOwnersEmails}` instead to get the values of the co-owners names and emails from the actual course.